pyproject.toml

```diff
@@ -24,10 +24,10 @@ classifiers = [
 [tool.poetry.dependencies]
 # pycollada is not working in Python 3.9. See
 # https://github.com/lace/tri-again/issues/25
-python = ">=3.7,<3.9"
+python = ">=3.7,<4"
 numpy = "*"
-polliwog = ">=2.0.0,<=3.0.0"
-pycollada = ">=0.7.1,<0.8"
+polliwog = {version = ">=2.0.0", allow-prereleases = true}
```

Comment:
Is this `allow-prereleases` OK?  Not sure how else to do it, since we want to bump lacecore here.

Reply:
Could you see what we are doing in other projects which are allowed to use polliwog prereleases?

Reply:
Well, I guess it's a moot issue now for this since we're requiring 3.0.0a0 or higher.  
In other instances we're doing something like `polliwog = ">=3.0.0a3"`  But if we wanted the lower bound start at 2.0.0, and the upper bound to be 3.0.0 but include the alphas before that, I think we'd need to do something like this.